docs/index.md

```diff
@@ -0,0 +1,105 @@
+# Welcome to the Uh!ive Python SDK
+
+The Uh!ive Python SDK is a library to access our live Automated Speech Recognition online APIs.
+It provides [I/O Free](https://sans-io.readthedocs.io/index.html) Python abstractions over the underlying protocols and workflows to hide the complexity.
+
+By providing an I/O Free implementation, we let developers choose whatever websocket transport library and paradigm — synchronous or asynchronous (asyncio) — they like most.
+
+## Access to the API
+
+In order to have access to our online APIs, your company needs to register for an account. Depending on the plan, you may get two kinds of credentials:
+
+* Either a `client_id` and `client_secret`;
+* or a `client_id`, `user_id` and `user_password`.
+
+In all cases, those credentials are used to retrieve a one time access token from our SSO.
+
+You are free to use whatever HTTP client library you like.
+
+Here is a synchronous example using `requests`:
+
+```python
+from uhlive.auth import build_authentication_request
+import requests
+
+uhlive_client = "…"
+uhlive_secret = "…"
+# user_id = "…"
+# user_password = "…"
+
+auth_url, auth_params = build_authentication_request(uhlive_client, uhlive_secret)
+# or auth_url, auth_params = build_authentication_request(uhlive_client, user_id=user_id, user_pwd=user_password)
```

Comment:
this is the old auth way? shouldn't we get rid of it as it is deprecated?

Reply:
both line 30 and 31 are KC connection? If so, ignore my comment

Reply:
both are KC connections indeed.